config/configtls/configtls.go

```diff
@@ -65,10 +69,15 @@ type TLSSetting struct {
 	ReloadInterval time.Duration `mapstructure:"reload_interval"`
 }
 
-// TLSClientSetting contains TLS configurations that are specific to client
+// TSLClientSetting contains TLS configurations that are specific to client
+// connections in addition to the common configurations.
+// Deprecated: [v0.96.0] Use ClientConfig instead.
+type TLSClientSetting = ClientConfig
+
+// ClientConfig contains TLS configurations that are specific to client
 // connections in addition to the common configurations. This should be used by
 // components configuring TLS client connections.
-type TLSClientSetting struct {
+type ClientConfig struct {
 	// squash ensures fields are correctly decoded in embedded struct.
 	TLSSetting `mapstructure:",squash"`
```

Comment:
Will you be able to handle this in a separate `breaking` PR?

Reply:
Sure. Will do that when I get a chance. Although, please feel free to take-over/reassign.